app.js

```diff
@@ -1,18 +1,48 @@
 'use strict';
 
+const fs = require('fs');
+const path = require('path');
 const Logger = require('./lib/logger.js');
 const SetupApp = require('./lib/setupApp');
 const { version } = require('./package.json');
 
 const serverPromise = SetupApp();
 
+const cwd = process.cwd();
+const NODE_ENV = process.env.NODE_ENV || 'development';
+const CONFIG_BASE_PATH = process.env.CONFIG_BASE_PATH || './config';
+const configFileName = `${NODE_ENV}.json`;
+
+let configPath;
+if (path.isAbsolute(CONFIG_BASE_PATH)) {
+  configPath = path.normalize(CONFIG_BASE_PATH) + path.sep + configFileName;
+} else {
+  configPath = CONFIG_BASE_PATH + '/' + configFileName;
+}
```

Comment:
Is this necessary? If we really have to validate the config, I'd rather we check the values of what `exp-config` returns than just checking whether the config file exists since the values are more important.

I suggest removing this file validation.

Reply:
To remove block. We won't be using config files instead as per discussion.